mmdet/models/dense_heads/paa_head.py

```diff
@@ -156,7 +154,10 @@ def loss(self,
             & (labels < self.background_label)).nonzero().reshape(-1)
 
         losses_cls = self.loss_cls(
-            cls_scores, labels, labels_weight, avg_factor=num_pos)
+            cls_scores,
+            labels,
+            labels_weight,
+            avg_factor=np.max([num_pos, len(img_metas)]))
```

Comment:
do the same weighting

Reply:
Add a comment to explain why max([num_pos, len(img_metas)])